### ISSUE TITLE
Allow filter by resolution

### ISSUE THREAD

**Issue by armandobs14** [calo001/fondo#14]

**Comment by MrDoodlezz** [comment]
An option to select the **preferred image orientation** (landscape or portrait) would also be super nice. Especially as a regular user one usually works with a **monitor in landscape orientation** - it is nice to see appealing, well photographed photographs, but _disappointing_ and not in the sense of the photographer that they are scaled during the selection or even cut to the format, because they were taken as portrait photos.

I'm no programmer by far, but shouldn't it be relatively easy to use a filter in the form of a button with an equation like _width > height = landscape_ and _height > width = portrait_ behind it?

Cheers
MrDoodlezz

**Comment by simonbcn** [comment]
Yes, bearing in mind that this is an application to change the wallpaper makes a lot of sense this filter or, at least, to be able to indicate the aspect ratio.
